source/CommonJobs/CommonJobs.Application.EvalForm/Commands/GetEvaluationCalifications.cs

```diff
@@ -0,0 +1,72 @@
+﻿using CommonJobs.Application.EvalForm.DTOs;
+using CommonJobs.Domain.Evaluations;
+using CommonJobs.Infrastructure.RavenDb;
+using Raven.Client.Linq;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+namespace CommonJobs.Application.EvalForm.Commands
+{
+    public class GetEvaluationCalifications : Command<CalificationsDTO>
+    {
+        private string _period { get; set; }
+
+        private string _evaluatedUser { get; set; }
+
+        private string _loggedUser { get; set; }
+
+        public GetEvaluationCalifications(string period, string evaluatedUser, string loggedUser)
+        {
+            _period = period;
+
+            _loggedUser = loggedUser;
+
+            _evaluatedUser = evaluatedUser;
+        }
+
+        public override CalificationsDTO ExecuteWithResult()
+        {
+            var evId = EmployeeEvaluation.GenerateEvaluationId(_period, _loggedUser);
+            var evaluation = RavenSession.Load<EmployeeEvaluation>(evId);
+
+            if (evaluation == null)
+            {
+                throw new ApplicationException(string.Format("Error: Evaluación inexistente: {0}.", evId));
+            }
+
+            RavenQueryStatistics stats;
+            var califications = RavenSession
+                .Query<EvaluationCalification>()
+                .Statistics(out stats)
+                .Customize(x => x.WaitForNonStaleResultsAsOfLastWrite())
+                .Where(x => x.EvaluationId == _period).ToList();
+
+            var calificationsToReturn = new List<EvaluationCalification>();
+
+            if (evaluation.ReadyForDevolution) // Ready to devolution
```

Comment:
Small typo: the comment still says "ready to devolution" when it should be "ready for devolution". (Thanks for the change in the enum!)